podcasts/Kids Profile/KidsProfileSheet.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+
+struct KidsProfileSheet: View {
+    @ObservedObject var viewModel: KidsProfileSheetViewModel
+
+    @EnvironmentObject var theme: Theme
+
+    var body: some View {
+        VStack {
+            if viewModel.currentScreen == .submit {
+                submitScreen
+                    .transition(viewModel.transition)
```

Comment:
I'm just curious: the `ViewModel` is, well, a view model, but why have the transition there? I feel like this is specific to this view.

No need to change, just curious about the reasoning behind it!

Reply:
@leandroalonso I thought to keep the view as much isolated as possible, but I moved the transition to the sheet container instead